lib/features/auto-subscribe.js

```diff
@@ -60,7 +87,10 @@ module.exports = ({
           }
         });
 
-        return Promise.all(promisedSubscriptions);
+        return Promise.all([
+          ...promisedSubscriptions,
+          _listenOnSyncEvents({rtm, filterFn}),
```

Comment:
What do you think of:

```js
_listenOnSyncEvents({rtm, filterFn});

return Promise.all([
    ...promisedSubscriptions
]);
```

In my opinion the event listening return values are out of interest.